crates/autopilot/src/decoded_settlement.rs

```diff
@@ -112,23 +112,30 @@ pub struct Order {
     pub buy_token: H160,
     pub sell_amount: U256,
     pub buy_amount: U256,
+    pub executed_amount: U256,
```

Comment:
We also have an `executed_fee_amount` part of the order which would avoid the fee computation 👇.

Reply:
I don't get the same results as in solver competition endpoint when I use `executed_fee_amount`. I will double check.

Reply:
So AFAIU, for objective value computation we use solver fees `Trade::executed_solver_fee()` while the `executed_fee_amount` you mentioned is based on the regular fee `Trade::executed_fee()` which are different so I can't use that.

Reply:
Makes sense.